src/AppBundle/Repository/PostRepository.php

```diff
@@ -38,8 +38,9 @@ public function findLatest($page = 1)
     {
         $query = $this->getEntityManager()
             ->createQuery('
-                SELECT p, t
+                SELECT p, a, t
                 FROM AppBundle:Post p
+                JOIN p.author a
```

Comment:
@yceruto, I think you're wrong. See http://docs.doctrine-project.org/projects/doctrine-orm/en/latest/reference/dql-doctrine-query-language.html#joins

> A join (be it an inner or outer join) becomes a “fetch join” as soon as fields of the joined entity appear in the SELECT part of the DQL query outside of an aggregate function.